MAINTAINERS.yml

```
@@ -241,6 +240,28 @@ Bluetooth controller:
     - "area: Bluetooth Controller"
     - "area: Bluetooth"
 
+Bluetooth host:
+  status: maintained
+  maintainers:
+    - jhedberg
+  collaborators:
+    - hermabe
+    - jori-nordic
+    - alwa-nordic
+    - Vudentz
+    - Thalley
+    - asbjornsabo
+    - sjanc
+    - theob-pro
+    - kruithofa
+  files:
+    - subsys/bluetooth/host/
+    - tests/bluetooth/host*/
+    - tests/bsim/bluetooth/host/
+  labels:
+    - "area: Bluetooth host"
```

FWIW, the label is `area: Bluetooth Host` with uppercase H.

```suggestion
    - "area: Bluetooth Host"
```

---

> FWIW, the label is `area: Bluetooth Host` with uppercase H.

@kruithofa will you make this change?